### DISCUSSION TITLE
How to remove the background color of nvdash?

### DISCUSSION THREAD

**Discussion by BrikeX** [NvChad/NvChad#2445] [Q&A] [answered]
In the Showcase part of README, the left figure does not have the blue background color. Please help me to remove that default background color. Thank you very much!
<img width="893" alt="image" src="https://github.com/NvChad/NvChad/assets/42525829/c41d6878-d149-4bea-8d6f-fb3e1eaa733c">

**Comment by siduck** [comment]
@BrikeX override hl groups ,   read nvchad.com pls

**Comment by Aoshisen** [comment, marked as answer]
~/.config/nvim/lua/custom/highlight.lua
<img width="736" alt="image" src="https://github.com/NvChad/NvChad/assets/66982800/6ee017e6-9f11-472d-88ae-367e5587c4b5">

**Comment by joaov-007** [comment]
nice 👍, I´m would opened  this issue

**Reply by siduck** [reply]
@joao-victor-lab i dont think so, you just have to override its highlight group

**Reply by joaov-007** [reply]
yup I do this works very well, good job!
